user_manual/universal_access.rst

```diff
@@ -0,0 +1,83 @@
+================
+Universal access
+================
+
+Universal access is very important to us. We follow web standards and check to
+make everything usable also with keyboard and assistive software such as
+screen readers. We aim to be compliant with the `Web Content Accessibility
+Guidelines 2.1 <https://www.w3.org/WAI/standards-guidelines/wcag/>`_
+on AA level, with the high contrast theme even on AAA level.
+We also follow the German BITV 2.0 guidelines.
+
+If you find any issues, don’t hesitate to report them on `our issue tracker
+<https://github.com/nextcloud/server/issues/>`_. And if you want to get
+involved, come `join our design team <https://nextcloud.com/design>`_!
+
+
+Zoom and responsiveness
+-----------------------
+
+The Nextcloud interface is fully responsive and usable on any size of screen.
+You can zoom in and out to fit the text and element size to your liking.
+The left navigation and right sidebar can be expanded or collapsed.
+
+
+Navigating via keyboard
+-----------------------
+
+You can navigate the web interface with keyboard only just like you can with the mouse:
+
+* ``Tab`` and ``Shift Tab`` to move between elements
+* ``Enter`` or ``Space`` to activate or open the element (depending on the type of element)
+* ``Escape`` to be used to close modals, popover menus, and file viewers
+* ``left arrow`` and ``right arrow`` to navigate between photos in the viewer
+* ``Ctrl f`` to focus the search field
+* ``Ctrl s`` to save changes in editors like Nextcloud Text
+
+For quicker navigation, we offer 2 "skip links" at the beginning of the document which allow you to:
+
+* Skip to main content
+* Skip to navigation of app
+
+Nextcloud Talk has shortcuts which are also documented inside the settings of the app itself:
+
+* ``c`` to focus the message input field
+* ``Escape`` to unfocus the message input field to be able to use shortcuts
+* ``f`` to fullscreen the chat or call
+* While in a call:
+   * ``m`` to toggle the microphone on and off
+   * ``v`` to toggle video on and off
+   * ``Space`` for push to talk or push to mute
+   * ``r`` to raise or lower hand
+
+Nextcloud Mail has shortcuts as well, also documented inside the settings of the app itself:
+
+* ``c`` to compose a new message
+* ``Left arrow`` to switch to a newer message
+* ``Right arrow`` to switch to an older message
+* ``s`` to toggle a message as favorite
+* ``u`` to toggle a message unread
```

Comment:
```suggestion
* ``U`` to toggle a message unread
```

Reply:
@Valdnet Why uppercase? See all other documents pleas.

Reply:
@rakekniven In my opinion, keyboard shortcuts should be capitalized. Just like it is in applications.

> See all other documents please.

If it is lowercase in other documents, this should also be changed 😉.

Reply:
Congratulations. You just found your next project 😄 .

Reply:
I guess it looks like it  :stuck_out_tongue_closed_eyes:.

Reply:
Right – I was looking at https://docs.github.com/en/github/getting-started-with-github/keyboard-shortcuts where they lowercase everything but it indeed looks weird compared to https://support.apple.com/en-us/HT201236

Will change it as per your suggestions, as I also double checked the app settings and they use upper case as well.

Reply:
And fixed :)